src/c++/examples/image_client.cc

```diff
@@ -135,7 +135,7 @@ Preprocess(
     if (img_channels == 1) {
```

Comment:
Please update the copyright from 
"// Copyright (c) 2020-2021, NVIDIA CORPORATION. All rights reserved."
to 
"// Copyright 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved."

PS: Did you validate these changes with image_client to ensure the model prediction is correct?

Reply:
Thanks for reviewing. Updated.

I ran these with the development container to ensure the tests pass. However, it looks like these aren't picked up in Gitlab. Is there an alternate way to test client files, or do I need to rely on local tests?

Reply:
You will need to create a Gitlab branch with the same name to trigger the CI. Lets continue this conversation on our channel

Reply:
@dyastremsky could you cleanup the commit history? There appear to be redundant commits

Reply:
Done! Thanks. :)